test/controllers/browse_controller_test.rb

```diff
@@ -60,33 +60,27 @@
 
       describe "with the new navigation enabled" do
         it "returns the original version of the page as the A variant" do
-          with_new_navigation_enabled do
-            with_variant EducationNavigation: "A" do
-              get :show, top_level_slug: "education"
+          with_new_navigation_disabled do
```

Comment:
There are still a few inconsistencies - the `describe` now contradicts the `with_new_navigation_enabled` on line 71, and the difference is between `with the new navigation not enabled` on line 48 and `with the new navigation enabled` on line 61.

The same thing applies to the other controller tests below.